Automattic/jetpack
# Add site options for WPCOM + GitHub integration

Related https://github.com/Automattic/wp-calypso/issues/70380

## Proposed changes:

* Add three new site options required for the WPCOM + GitHub integration. 
  - `wpcom_github_hosting_repo_name`
  - `wpcom_github_hosting_branch_name`
  - `wpcom_github_hosting_base_path`


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Follow the steps to apply this patch to your sandbox by using the instructions below in a separate comment. 
* Go to `/hosting-config/:site` for a site where you have [connected GitHub](https://github.com/Automattic/wp-calypso/pull/73030) and you should see the values that were saved when you hit "Connect" if you check the `sites.options` in the response of `/rest/v1.2/sites/:site`. 
* Review the code: Did I add the code to the correct place? Do we even need a change log entry?
